packages/common/src/projects/_internal/ProjectUiSchemaEdit.ts

```diff
@@ -281,6 +281,60 @@ export const buildUiSchema = async (
           },
         ],
       },
+      {
+        type: "Section",
+        labelKey: `${i18nScope}.sections.embeds.label`,
+        options: {
+          headerTag: "h2",
+          helperText: { labelKey: `${i18nScope}.sections.embeds.helperText` },
+        },
+        elements: [
+          {
+            scope: "/properties/view/properties/embeds",
+            type: "Control",
+            options: {
+              control: "hub-composite-input-embeds",
+              facets: [
+                {
+                  label: `{{${i18nScope}.fields.embeds.facets.type.label:translate}}"`,
```

Comment:
minor: I think you have an extra " at the end of all of these facet labels - irrelevant if you end up moving these to the field itself